test/selenium/lib/testPrepperSelenium.js

```diff
@@ -5,19 +5,18 @@ var foxboxManager = require('./foxboxHelperSelenium.js');
 var testPrepperSelenium = (function() {
 
   function beforeTest(done) {
-
     console.log('test started');
     foxboxManager.fullOptionStart(done);
   }
 
   function makeSuite(desc, test) {
     describe(desc, function () {
-      this.timeout(5000000);
+      this.timeout(30000);
       before(beforeTest);
```

Comment:
I am currently in process of separating the sim startup routine so we can customize which sim to start up (needed for the hue authentication test), will ask for review once it's done